markdown-mode.el

```diff
@@ -928,6 +928,12 @@ Group 3 matches the mathematical expression contained within.
 Group 2 matches the opening slashes, and is used internally to
 match the closing slashes.")
 
+(defconst markdown-regex-escaped-char
+  "\\(?1:\\\\\\)\\(?2:[-~_.*\\\\]\\)"
```

Comment:
https://spec.commonmark.org/0.30/#backslash-escapes

I'm not sure that are escaped characters enough ? CommonMark specification says following

> Any ASCII punctuation character may be backslash-escaped